package.json

```diff
@@ -14,8 +14,9 @@
     "prepare": "husky install",
     "test": "jest",
     "vercel-build": "./vercel.sh",
-    "build:preview": "prisma generate && echo \"y\" | prisma db push && next build",
-    "build:main": "prisma generate & next build"
+    "build:preview": "prisma generate && echo \"y\" | echo \"y\" | prisma db push && next build",
+    "build:main": "prisma generate & next build",
+    "startup": "docker-compose -f docker-compose.yml up -d && next dev"
```

Comment:
This script should delegate to the ``db:start`` script. 

``"startup": "yarn db:start && next dev"``